src/main/java/org/isf/menu/gui/MainMenu.java

```diff
@@ -92,15 +86,17 @@ public void commandInserted(AWTEvent e) {
 
 	public static boolean checkUserGrants(String code) {
 
-		for (UserMenuItem umi : myMenu) {
+		Iterator<UserMenuItem> it = myMenu.iterator();
+		while (it.hasNext()) {
+			UserMenuItem umi = it.next();
```

Comment:
correct form was with implicit iterator

Reply:
I restored the current informatici version